exercises/practice/armstrong-numbers/tests/armstrong-numbers.rs

```diff
@@ -52,3 +52,27 @@ fn test_seven_digit_armstrong_number() {
 fn test_seven_digit_non_armstrong_number() {
     assert!(!is_armstrong_number(9_926_316))
 }
+
+#[test]
+#[ignore]
+fn test_nine_digit_armstrong_number() {
+    assert!(is_armstrong_number(912_985_153));
+}
+
+#[test]
+#[ignore]
+fn test_nine_digit_non_armstrong_number() {
+    assert!(!is_armstrong_number(999_999_999));
+}
+
+#[test]
+#[ignore]
+fn test_ten_digit_non_armstrong_number() {
+    assert!(!is_armstrong_number(3_999_999_999));
+}
+
+#[test]
+#[ignore]
+fn test_ten_digit_fake_armstrong_number() {
```

Comment:
let's make it clearer what "fake" means - your commit message already explains it may appear as an armstrong number under `wrapping_add`, but this commit message wouldn't be visible to a student solving this exercise without knowledge of this repo. So I would think this test should be renamed or a comment should be added

Reply:
Done.